projects/security.md

```diff
@@ -0,0 +1,58 @@
+# Security Semantic Conventions Working Group
+
+## Description
+
+The purpose of this working group is to bring in the security domain for the OpenTelemetry community. 
+
+As outlined in the [ECS OTEP](https://github.com/open-telemetry/oteps/blob/main/text/0199-support-elastic-common-schema-in-opentelemetry.md), the Elastic Common Schema (ECS) is currently being contributed to the semantic conventions schema. Given the significance of security within ECS, establishing this working group is crucial as it will expedite the donation of ECS fields tailored to security use cases. Beyond expanding the schema, our aim is to craft a clear vision for the instrumentation required.
```

Comment:
Can we have a word here on why security wasn't part of the initial donation and why it makes sense to have it _now_?

Reply:
Security was always a part of initial donation. We need this group so we could proceed with security namespaces/fields and so that there are other domain experts who would work with us on that matter